src/EventHub.Domain/Organizations/Plans/PlanFeatureManager.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Threading.Tasks;
+using EventHub.Events;
+using EventHub.Events.Registrations;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Domain.Repositories;
+
+namespace EventHub.Organizations.Plans;
+
+public class PlanFeatureManager : ITransientDependency
+{
+    private readonly IPlanInfoDefinitionStore _planInfoDefinitionStore;
+    private readonly IRepository<Organization, Guid> _organizationRepository;
+    private readonly IRepository<Event, Guid> _eventRepository;
+    private readonly IEventRegistrationRepository _eventRegistrationRepository;
+
+    public PlanFeatureManager(
+        IPlanInfoDefinitionStore planInfoDefinitionStore, 
+        IRepository<Organization, Guid> organizationRepository, 
+        IRepository<Event, Guid> eventRepository, 
+        IEventRegistrationRepository eventRegistrationRepository)
+    {
+        _planInfoDefinitionStore = planInfoDefinitionStore;
+        _organizationRepository = organizationRepository;
+        _eventRepository = eventRepository;
+        _eventRegistrationRepository = eventRegistrationRepository;
+    }
+    
+    public virtual async Task<bool> CanCreateNewEvent(Guid organizationId)
+    {
+        var currentPlanOfOrganization = await GetCurrentPlanByOrganizationIdAsync(organizationId);
+        
+        var totalEventCountByOrganization = await _eventRepository.CountAsync(x => x.OrganizationId == organizationId && x.CreationTime.Year < DateTime.Now.Year + 1 && x.CreationTime.Year > DateTime.Now.Year - 1);
+        totalEventCountByOrganization++;
+        return currentPlanOfOrganization.Feature.MaxAllowedEventsCountInOneYear >= totalEventCountByOrganization;
+    }
+    
+    public virtual async Task<bool> CanAddNewTrack(Event @event)
+    {
+        var currentPlanOfOrganization = await GetCurrentPlanByOrganizationIdAsync(@event.OrganizationId);
+
+        var tracksCount = @event.Tracks.Count;
+        tracksCount++;
+        return currentPlanOfOrganization.Feature.MaxAllowedTracksCountInOneEvent >= tracksCount;
```

Comment:
```suggestion
        return currentPlanOfOrganization.Feature.MaxAllowedTracksCountInOneEvent > @event.Tracks.Count;
```

Would be simpler :))